### DISCUSSION TITLE
Join meetup?

### DISCUSSION THREAD

**Discussion by EvaMaeRey** [ggplot2-extenders/ggplot-extension-club#35] [General]
Please drop a reply or comment if you'd like to join the virtual meetup.

**Comment by teunbrand** [comment]
I can join the 19th of August

**Reply by EvaMaeRey** [reply]
Good to know!!  I actually opened to complement a more public-facing version of the repo (a website - probably github pages.)  And then link to this discussion so people can say they want to be added to mailing list.  Thoughts?   


proposed home page:

![image](https://github.com/user-attachments/assets/4f786659-9b77-4cb2-b86f-8eba781deec9)

'Meeting' page w/ link to this discussion thread:

![image](https://github.com/user-attachments/assets/87bcb383-fd7a-468e-aa05-d5d58746457e)

maybe a form would be better.  

Thoughts in general about pages version of repo material?

**Reply by yjunechoe** [reply]
A website is a great idea - thanks for putting it together!

I think a form (google forms?) would be a bit more inviting than github. Plus then we also get to ask some questions first like their timezone, if/what extensions they've developed, if they'd be willing to present in a future meeting, etc.

**Reply by EvaMaeRey** [reply]
Good ideas!  Thanks June!  The forms direction and being able to solicit a bit more info sounds better.  

After sending the last reply I went down a rabbit hole on 'equivocations' meaning.  I think it can be used this way as in flip-flop, waffle, vacillate, but doesn't seem like the first cited definition which is like misrepresent the truth or something like that.   Does it sound weird to you there?   Maybe just replace with 'doubts'.

**Reply by EvaMaeRey** [reply]
BTW, I think I don't have access to github pages (my preference), since @teunbrand owns this repo.  I'm not sure what would be way forward to manage.  I wanted to just work something up and make it live for you all to check out, but I forgot about permissions.

**Reply by teunbrand** [reply]
GH pages was setup correctly, it was just not linked to from anywhere 😅 I've now put the link in the description.

**Reply by EvaMaeRey** [reply]
Ah great!  I'm not a super website/github pages user but trying more in this space recently.

**Reply by EvaMaeRey** [reply]
I'm cross-linking with other web work that I'm doing. So from (and to?) geom recipes and the cookbook (like where to go for help if stack overflow isn't enough) but also from this umbrella page I'm working on... https://evamaerey.github.io/everyday_ggplot2_extension/community.html  Visual theme includes teapot (kind of a 'design of everyday things' nod) and ggplot2 hex in a field of hexes...  Thoughts?

![image](https://github.com/user-attachments/assets/5aecca6e-4efb-4a68-a524-37b17fd41125)
